frontend/mulighetsrommet-veileder-flate/src/components/filtrering/Filtertags.tsx

```diff
@@ -36,7 +36,7 @@ export function Filtertags({ filter, setFilter }: FiltertagsProps) {
     filter.lokasjoner.length > 0;
 
   return (
-    <div className={styles.filtertags} data-testid="filtertags">
+    <div className={styles.filtertags} data-testid="filtertags" id="filtertags">
```

Comment:
Kan du ikke gjenbruke data-testid-attributet istedenfor å lage en egen for id? 
På react-joyride dokumentasjonen står det at selector kan være en css-selector, så da tror jeg du skal kunne bruke uthenting basert på data-attributt istedenfor. 

Her står det om css selectors med data-attributer https://css-tricks.com/almanac/selectors/a/attribute/